LayoutTests/fast/canvas/canvas-filter-font-relative-unit.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<body>
+    <canvas id="canvas"></canvas>
+    <p>This test passes if it does not crash</p>
+
+    <script>
+        if (window.testRunner) {
+            testRunner.waitUntilDone();
```

Comment:
Why do we have to `waitUntilDone()`? I think this test is correct if it is synchronous.

Reply:
I copied the test from a different test that crashes the GPU process. The wait is to allow WebKitTestRunner to catch the GPU process crash. I'll remove it.

Reply:
Using `setInterval()` makes the test fragile. If you want to wait _sometime_, I think you can wait for specific number of frames using `requestAnimationFrame()`.

Reply:
I didn't know that trick, thanks! For this test we don't have to wait so I'll remove the `setInterval` regardless, but for when the test has to wait, I'll use `requestAnimationFrame()` and count frames.